src/resources/swagger.yaml

```diff
@@ -55,7 +55,17 @@ paths:
                     type: string
                   code:
                     type: string
-                    enum: ["TSV_PARSING_FAILED"]
+                    enum: ["TSV_PARSING_FAILED", "INVALID_FILE_ERROR"]
```

Comment:
you also need to add `batchName` to  the registration schema : #/components/schemas/Registration